README.md

```diff
@@ -397,6 +397,16 @@ test "render component" do
 end
 ```
 
+Alternatively, assert against the raw output of the component, which is exposed as `rendered_component`:
```

Comment:
I'd be cool with replacing the example above (with `result = render_inline`) with this example. I think it's a much nicer approach 😄

Reply:
The benefit of the other approach is that you can use CSS selectors to make more targeted assertions about certain elements in the rendered output (without relying on Capybara). So I still think there is some value in documenting it.